pkg/webhook/virtualmachine/virtualmachine.go

```diff
@@ -66,7 +66,7 @@ func (a *virtualMachineAnnotator) Handle(ctx context.Context, req admission.Requ
 		virtualMachine.Namespace = req.AdmissionRequest.Namespace
 	}
 
-	log.V(1).Info("got a create virtual machine event", "virtualMachineName", virtualMachine.Name, "virtualMachineNamespace", virtualMachine.Namespace)
+	log.V(1).Info("got a virtual machine event", "virtualMachineName", virtualMachine.Name, "virtualMachineNamespace", virtualMachine.Namespace)
```

Comment:
How about moving this to line 71 and create another one for update events in line 77?

Reply:
[mutateCreateVirtualMachinesFn](https://github.com/k8snetworkplumbingwg/kubemacpool/blob/9494a62561c713192706d0c23b14b70dfcdff2a3/pkg/webhook/virtualmachine/virtualmachine.go#L95) and [mutateUpdateVirtualMachinesFn](https://github.com/k8snetworkplumbingwg/kubemacpool/blob/9494a62561c713192706d0c23b14b70dfcdff2a3/pkg/webhook/virtualmachine/virtualmachine.go#L118) alreaddy have prints of their own. maybe this line is irrelevant?